Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -11,10 +11,9 @@ protocol PledgeViewControllerDelegate: AnyObject {
 final class PledgeViewController: UIViewController, MessageBannerViewControllerPresenting {
   // MARK: - Properties
 
-  private lazy var confirmButton: UIButton = { UIButton(type: .custom) }()
```

Comment:
Yeah this will really mess up my loading button but it should be fairly simple to fix